src/objects.ts

```diff
@@ -0,0 +1,73 @@
+import { makeEnum } from '@transcend-io/type-utils';
+
+/** The purpose of processing data */
+export const ProcessingPurpose = makeEnum({
```

Comment:
Just a note, if we will be using this same enum on our backend, the order of the values of this enum (and the one below) will be the order in which the options display when viewing a dropdown for the purpose and category.

Reply:
sounds good 👍 if we eventually translate those enums and make a `defineMessages` object, we could define the order there as well